app/posts/apply-for-teacher-training/2020-01-20-give-a-reference-iteration.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Give a reference iteration
+description:
+tags:
+---
+
+## User needs
+
+As a referee
+I need to provide a useful reference
+So that I can help the candidate with their application and get back to my day as quickly as possible
+
+As a candidate
+I need to know that my referee doesn't want to give a reference
+So that I can get another referee as quickly as possible to progress my application
+
+## Hypotheses
+
+### Actively refusing to give a reference
+
+Some referees may not want to, or be able to give a reference. This slows down the candidate's application process.
+
+**If** we let users tell us that they won't give a reference
+**Then** we'll be able to notify the candidate and get another referee quickly
+**We'll know this works when**:
+- Referees click the refuse link in the email and confirm their refusal
+
+### One big text box for the reference
+
+We don't have any evidence that referees and providers need anything more sophisticated or bespoke than a single text box. So for now, we're just keeping parity with UCAS.
+
+We're also wary about asking directly if the candidate is safe to work with children. Providers like this, but we feel we need to be careful how we ask this. The absence of the flag might be seen as a bad when really the referee didn't know either way.
+
+We also don't have enough confidence in the 1 to 10 scales, which providers described as ‘subjective’. A single box is the lowest risk.
+
+**If** we let users enter everything into 1 big text box
+**Then** we'll be able to see what problems arise with referees and providers
```

Comment:
Maybe: 
We'll know this works when providers successfully use the references given to make decisions